src/tracks/playabletrack/wavetrack/ui/WaveformCache.cpp

```diff
@@ -11,23 +11,15 @@
 #include "WaveformCache.h"
 
 #include <cmath>
+#include <cstring>
+
 #include "Sequence.h"
 #include "GetWaveDisplay.h"
 #include "WaveClipUtilities.h"
 
 class WaveCache {
 public:
-   WaveCache()
-      : dirty(-1)
-      , start(-1)
-      , pps(0)
-      , rate(-1)
-      , where(0)
-      , min(0)
-      , max(0)
-      , rms(0)
-   {
-   }
+   WaveCache() = default;
```

Comment:
The initialization of `pps` changed from 0 to -1.  Does it matter?

Reply:
I don't think so, but let's revert for consistency